### PR TITLE
6.0.20RN updates

### PR DESCRIPTION
**Pull request by AlonMagrafta**

### CODE REVIEW

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 1]
```diff
@@ -149,6 +152,14 @@ All known bugs around setting ciphers were fixed.  To learn more, see [Updating
 
 -   Envoy updated and verified with multiple security headers.
 
+-   Starting Redis Software 6.0.20:
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:155]: When you say "starting" do you mean this is the behavior from 6.0.20 forward, or are you referring to how the customer starts the software?

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 2]
```diff
@@ -166,6 +177,8 @@ All known bugs around setting ciphers were fixed.  To learn more, see [Updating
 
 ## Known limitations
 
+-   RS55504 - Bug RS6.0.20-66 (Build #66) causing RediSearch cursor to break. Please upgrade to a higher build when running with RediSeach.
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:180]:
```suggestion
-   RS55504 - Bug RS6.0.20-66 (Build #66) causes RediSearch cursor to break. Please upgrade to a higher build when running with RediSeach. 
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 3]
```diff
@@ -149,6 +152,14 @@ All known bugs around setting ciphers were fixed.  To learn more, see [Updating
 
 -   Envoy updated and verified with multiple security headers.
 
+-   Starting Redis Software 6.0.20:
+
+    -   The replication backlog size of new databases will be allocated dynamically according to shard size.
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:157]:
```suggestion
    -   The replication backlog size of new databases is allocated dynamically according to shard size.
```

File: content/rs/release-notes/rs-6-0-20-april-2021.md (hunk 1 above)
Comment (AlonMagrafta) [reply content/rs/release-notes/rs-6-0-20-april-2021.md:155]: the behavior from 6.0.20 forward

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 4]
```diff
@@ -149,6 +152,14 @@ All known bugs around setting ciphers were fixed.  To learn more, see [Updating
 
 -   Envoy updated and verified with multiple security headers.
 
+-   Starting Redis Software 6.0.20:
+
+    -   The replication backlog size of new databases is allocated dynamically according to shard size.
+
+    -   The Active-Active replication backlog size of new Active-Active databases will be allocated dynamically according to shard size.
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:159]:
```suggestion
    -   The Active-Active replication backlog size of new Active-Active databases is allocated dynamically according to shard size.
```

File: content/rs/release-notes/rs-6-0-20-april-2021.md (hunk 1 above)
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:155]:
```suggestion
-   Starting with Redis Software 6.0.20:
```

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 5]
```diff
@@ -141,6 +141,9 @@ database](https://docs.redislabs.com/latest/modules/upgrading-rs/#upgrading-the-
 
     -   Sentinel discovery service
 
+-   Starting Redis Software 6.0.20 new clusters will be set with minimal TLS version v1.2
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:144]:
```suggestion
-   Starting with Redis Software 6.0.20, new clusters will be set with minimal TLS version v1.2
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**